File: AlgoWorks/src/com/solid/algolearning/javacode/algorithms/searching/StopWatch.java
package com.solid.algolearning.javacode.algorithms.searching;

public class StopWatch {
    private final long start;

    //Constructor
    public StopWatch(){
        start = System.currentTimeMillis();
    }

    /*Method to return the elapsed CPU time(in seconds) since the stopwatch was created
    */
    public double elapsedTime(){
        long now = System.currentTimeMillis();
        return (now - start) / 1000.0;
    }
}

File: AlgoWorks/src/com/solid/algolearning/kotlincode/algorithms/searching/StopWatch.kt
package com.solid.algolearning.kotlincode.algorithms.searching

class StopWatch {
    val start: Long = System.currentTimeMillis()

    fun elapsedTime(): Double{
        val now: Long = System.currentTimeMillis()
        return (now - start) / 1000.00
    }
}